Resources/Prototypes/Species/human.yml

```
@@ -40,7 +41,7 @@
   id: MobHumanMarkingLimits
   points:
     Special: # the cat ear joke
-      points: 0
+      points: 1 # Carpmosia-edit - Allow cat parts
```

```suggestion
      points: 0
```
This is a problem because cat ears are not in the ears category, letting people have both normal ears and cat ears

Probably better to add cat ears in a separate PR

---

I mean I can make them part of HeadTop but I don't think having them exclusive is necessary

---

Oh and having them be HeadTop changes how they'd layer

---

> Oh and having them be HeadTop changes how they'd layer

any issues regarding that?

---

> I mean I can make them part of HeadTop but I don't think having them exclusive is necessary

should be, they are very close to normal ears and would partially overlap